src/core/operators/PDFOperatorNames.ts

```diff
@@ -38,6 +38,7 @@ const enum PDFOperatorNames {
   SetRenderingIntent = 'ri',
 
   // Graphics Operators
+  AppendLine = 'ln',
```

Comment:
This new operator does not appear to be used. And I am unable to find it anywhere in the PDF spec. Do we need to keep this change, or can it be removed?

Reply:
Hi @Hopding ,

Yes, you are right, it can be removed. I added it because I thought it will be used in the generated PDF. I will remove it.